projects/aca-folder-rules/src/lib/rule-details/conditions/rule-condition-fields.ts

```diff
@@ -45,7 +45,7 @@ export const ruleConditionFields: RuleConditionField[] = [
   {
     name: 'mimetype',
     label: 'ACA_FOLDER_RULES.RULE_DETAILS.FIELDS.MIMETYPE',
-    type: 'string'
+    type: 'special'
```

Comment:
the value "special" seems a bit weird... as this is the mime type picker, I suggest calling it as what it is `mimeType`

Reply:
changes type